mpp/mpp.F90

```diff
@@ -230,6 +230,9 @@ module mpp_mod
   public :: read_ascii_file, read_input_nml, mpp_clock_begin, mpp_clock_end
   public :: get_ascii_file_num_lines
   public :: mpp_record_time_start, mpp_record_time_end
+  !! TODO:
+  public :: mpp_init_logfile
```

Comment:
Are you still working on this?  Are you making a private routine public?

Reply:
I was hoping to not make it public. Is there a better way to get access to mpp_init_logfile from the unit test app.

Reply:
This may remain a problem (with both pros and cons) see the issue below concerning  mpp_init_test_logfile_init

Reply:
Thanks for bringing up the issue in the meeting. Fortunately for this test is a non-issue now.